src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public enum Colors {
+    Black,
+    White,
+    Green,
+    Yellow,
+    Orange,
+    Red,
+    Blue,
+    Brown,
+    Purple
```

Comment:
UPPER CASE for constants
```suggestion
    Black,
    White,
    Green,
    Yellow,
    Orange,
    Red,
    Blue,
    Brown,
    Purple
```